qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteDiagnostics.java

```diff
@@ -530,6 +530,14 @@ private ResolvedJavaTypeInfo validateObjectPart(ObjectPart objectPart, Section o
 				// The data model is not loaded, ignore the error of undefined variable
 				return null;
 			}
+			QuteProject project = template.getProject();
+			if (project != null) {
+				if (project.isUserTag(template)) {

```

Comment:
can be combined in a single if statement

Reply:
fixed